tutorials/workshop/Workshop_1_Task_Intro.ipynb

```diff
@@ -0,0 +1,339 @@
+       "      <td>[,, 26, ,, who]</td>\n",
+       "      <td>[s, ,, with, the]</td>\n",
+       "      <td>[,, 26, ,, who, looked, chic, in, a, very, fem...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>(0, 0)</td>\n",
+       "      <td>(4, 4)</td>\n",
+       "      <td>Richards and her sister Kyle have been regular...</td>\n",
+       "      <td>[Richards, and, her, sister, Kyle, have, been,...</td>\n",
+       "      <td>[and, her, sister, Kyle]</td>\n",
+       "      <td>[have, been, regular, cast]</td>\n",
+       "      <td>[and, her, sister]</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  person1_word_idx person2_word_idx  \\\n",
+       "5           (3, 4)           (8, 9)   \n",
+       "6           (0, 0)           (2, 3)   \n",
+       "7         (44, 45)         (51, 51)   \n",
+       "8         (11, 12)         (32, 32)   \n",
+       "9           (0, 0)           (4, 4)   \n",
+       "\n",
+       "                                            sentence  \\\n",
+       "5  Us Weekly reported Andy Cohen and Hanks’ wife ...   \n",
+       "6  Williams and Ven Veen arrived at the ranch ear...   \n",
+       "7  As Agrecovery now offers a recycling solution ...   \n",
+       "8  Joining him on the red carpet was his co-star ...   \n",
+       "9  Richards and her sister Kyle have been regular...   \n",
+       "\n",
+       "                                              tokens  \\\n",
+       "5  [Us, Weekly, reported, Andy, Cohen, and, Hanks...   \n",
+       "6  [Williams, and, Ven, Veen, arrived, at, the, r...   \n",
+       "7  [As, Agrecovery, now, offers, a, recycling, so...   \n",
+       "8  [Joining, him, on, the, red, carpet, was, his,...   \n",
+       "9  [Richards, and, her, sister, Kyle, have, been,...   \n",
+       "\n",
+       "        person1_right_tokens         person2_right_tokens  \\\n",
+       "5  [and, Hanks’, wife, Rita]   [,, who, plays, Williams’]   \n",
+       "6  [and, Ven, Veen, arrived]    [arrived, at, the, ranch]   \n",
+       "7     [, With, FIL, joining]     [is, now, supported, by]   \n",
+       "8            [,, 26, ,, who]            [s, ,, with, the]   \n",
+       "9   [and, her, sister, Kyle]  [have, been, regular, cast]   \n",
+       "\n",
+       "                                      between_tokens  \n",
+       "5                                [and, Hanks’, wife]  \n",
+       "6                                              [and]  \n",
+       "7                          [, With, FIL, joining, ,]  \n",
+       "8  [,, 26, ,, who, looked, chic, in, a, very, fem...  \n",
+       "9                                 [and, her, sister]  "
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dev_data[5:10]"
```

Comment:
dev_df.head()

Reply:
the first few were confusing so I changed it in a commit, ill go back to `.head()`